modules/local/liftoff/main.nf

```diff
@@ -0,0 +1,46 @@
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process LIFTOFF {
```

Comment:
This has a module on nf-core too, although you may want to integrate the tool Lifton instead

Reply:
Thanks for the suggestion, I will probably switch to lifton.